generator/templates/server.go.tmpl

```diff
@@ -777,9 +780,6 @@ return nil, err
 {{ .Resolver.ProtoComment }}
 */
 {
-	{{- if .Resolver.IsValidation }}
```

Comment:
I've moved it into the `callResolver` template

1. To reuse the code for the `if` statement
2. To use the `sg` variable from validations